README.adoc

```diff
@@ -1103,8 +1103,7 @@ User.where.not('status = ? AND plan = ?', 'active', 'basic')
 
 === Redundant `all` [[redundant-all]]
 
-Using `all` as a receiver for Active Record query methods is redundant. 
-The result won't change without `all`, so it can be removed.
+When the receiver for Active Record query methods is `all`, the result won't change if `all` is omitted.
```

Comment:
Can we consider ‘delete*’ “query methods”?
If not, we should emphasize on that.